test/records/test_image.py

```diff
@@ -46,8 +47,10 @@ def test_create_thumbnail(self):
         )
         test_image.create_thumbnail()
 
-        thumbnail_checksum = hashlib.md5(test_image.thumbnail).hexdigest()
-        assert thumbnail_checksum == "d4f018fe6ab61b1706244e95b1d9bb96"
+        bytes_thumbnail = base64.b64decode(test_image.thumbnail)
+        img = PILImage.open(BytesIO(bytes_thumbnail))
+        thumbnail_checksum = str(imagehash.phash(img))
+        assert thumbnail_checksum == "0000000000000000"
```

Comment:
Can you add a comment above this line to explain why we're asserting for a hash which contains all 0's please? I remember this is something that you asked about and it took me a few moments to work out why so a comment would likely be helpful to people in the future :)